src/accelerate/accelerator.py

```diff
@@ -2536,7 +2535,7 @@ def save_model(
         model: torch.nn.Module,
         save_directory: Union[str, os.PathLike],
         max_shard_size: Union[int, str] = "10GB",
-        safe_serialization: bool = False,
+        safe_serialization: bool = True,
```

Comment:
Would it be useful to have a deprecation cycle or can we assume that there is no risk for users when we change the default? Since this is intended for checkpointing, I assume it should be fine.

Reply:
There shouldn't be a risk, especially because of our fallback in case of different saving techniques